### ISSUE TITLE
[INSTALL] Cellpose GPU cuDNN error

### ISSUE THREAD

**Issue by felixlouchez** [MouseLand/cellpose#852]
**Install problem**
Hello,
I have installed the cellpose GUI using anaconda prompt on windows.
When using the cpu it is working well but it is very slow so I want to switch to the gpu of my computer (NVDIA Quadro K4200).
So following the instruction of github webpage of cellpose, I used this command line : conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia

and using “conda list” to check for pytorch i have the following line :

pytorch 2.2.0 py3.8_cuda11.8_cudnn8_0 pytorch

now when I run cellpose I can select “use gpu”, but when I try to run one of the model i have the following error :
“NET ERROR: cuDNN error: CUDNN_STATUS_NOT_INITIALIZED”

![error](https://github.com/MouseLand/cellpose/assets/159019428/ae5976f4-590d-46dc-8c67-c424977216d7)

I did some research and the compute capability of my gpu is 3.0. 
Cuda 11.X requires 3.5 or more and the cellpose github says that I should have a CUDA version > 10.2, so at least 11.0, does it mean there is no way I can run cellpose on my gpu ?

It is my first time installing something using a command prompt so I feel very lost. Thanks in advance.

**Environment info**

(cellpose) C:\Users\felouche>conda list
# packages in environment at C:\Users\felouche\AppData\Local\anaconda3\envs\cellpose:
#
# Name                    Version                   Build  Channel
_tflow_select             2.3.0                     eigen
abseil-cpp                20230802.0           h5da7b33_2
absl-py                   1.4.0            py38haa95532_0
aiohttp                   3.9.0            py38h2bbff1b_0
aiosignal                 1.2.0              pyhd3eb1b0_0
astor                     0.8.1            py38haa95532_0
astunparse                1.6.3                      py_0
async-timeout             4.0.3            py38haa95532_0
attrs                     23.1.0           py38haa95532_0
blas                      1.0                         mkl
blinker                   1.6.2            py38haa95532_0
brotli-python             1.0.9            py38hd77b12b_7
c-ares                    1.19.1               h2bbff1b_0
ca-certificates           2023.12.12           haa95532_0
cachetools                5.3.2                    pypi_0    pypi
cellpose                  2.2.3                    pypi_0    pypi
certifi                   2023.11.17       py38haa95532_0
cffi                      1.16.0           py38h2bbff1b_0
charset-normalizer        3.3.2                    pypi_0    pypi
click                     8.1.7            py38haa95532_0
colorama                  0.4.6            py38haa95532_0
cryptography              41.0.7           py38h89fc84f_0
cuda-cccl                 12.3.101                      0    nvidia
cuda-cudart               11.8.89                       0    nvidia
cuda-cudart-dev           11.8.89                       0    nvidia
cuda-cupti                11.8.87                       0    nvidia
cuda-libraries            11.8.0                        0    nvidia
cuda-libraries-dev        11.8.0                        0    nvidia
cuda-nvrtc                11.8.89                       0    nvidia
cuda-nvrtc-dev            11.8.89                       0    nvidia
cuda-nvtx                 11.8.86                       0    nvidia
cuda-profiler-api         12.3.101                      0    nvidia
cuda-runtime              11.8.0                        0    nvidia
cudatoolkit               10.2.89              h74a9793_1
fastremap                 1.14.1                   pypi_0    pypi
filelock                  3.13.1           py38haa95532_0
freetype                  2.12.1               ha860e81_0
frozenlist                1.4.0            py38h2bbff1b_0
fsspec                    2023.12.2                pypi_0    pypi
gast                      0.4.0              pyhd3eb1b0_0
giflib                    5.2.1                h8cc25b3_3
gmpy2                     2.1.2            py38h7f96b67_0
google-api-core           2.16.1                   pypi_0    pypi
google-auth               2.27.0                   pypi_0    pypi
google-auth-oauthlib      0.4.4              pyhd3eb1b0_0
google-cloud-core         2.4.1                    pypi_0    pypi
google-cloud-storage      2.14.0                   pypi_0    pypi
google-crc32c             1.5.0                    pypi_0    pypi
google-pasta              0.2.0              pyhd3eb1b0_0
google-resumable-media    2.7.0                    pypi_0    pypi
googleapis-common-protos  1.62.0                   pypi_0    pypi
grpc-cpp                  1.48.2               h6772dbd_4
grpcio                    1.48.2           py38h6772dbd_4
gtest                     1.14.0               h59b6b97_0
h5py                      2.10.0           py38h5e291fa_0
hdf5                      1.10.4               h7ebc959_0
icc_rt                    2022.1.0             h6049295_2
idna                      3.6                      pypi_0    pypi
imagecodecs               2023.3.16                pypi_0    pypi
importlib-metadata        7.0.1            py38haa95532_0
intel-openmp              2023.1.0         h59b6b97_46320
jinja2                    3.1.3                    pypi_0    pypi
jpeg                      9e                   h2bbff1b_1
keras-applications        1.0.8                      py_1
keras-preprocessing       1.1.2              pyhd3eb1b0_0
lerc                      3.0                  hd77b12b_0
libcublas                 11.11.3.6                     0    nvidia
libcublas-dev             11.11.3.6                     0    nvidia
libcufft                  10.9.0.58                     0    nvidia
libcufft-dev              10.9.0.58                     0    nvidia
libcurand                 10.3.4.107                    0    nvidia
libcurand-dev             10.3.4.107                    0    nvidia
libcusolver               11.4.1.48                     0    nvidia
libcusolver-dev           11.4.1.48                     0    nvidia
libcusparse               11.7.5.86                     0    nvidia
libcusparse-dev           11.7.5.86                     0    nvidia
libdeflate                1.17                 h2bbff1b_1
libffi                    3.4.4                hd77b12b_0
libjpeg-turbo             2.0.0                h196d8e1_0
libnpp                    11.8.0.86                     0    nvidia
libnpp-dev                11.8.0.86                     0    nvidia
libnvjpeg                 11.9.0.86                     0    nvidia
libnvjpeg-dev             11.9.0.86                     0    nvidia
libpng                    1.6.39               h8cc25b3_0
libprotobuf               3.20.3               h23ce68f_0
libtiff                   4.5.1                hd77b12b_0
libuv                     1.44.2               h2bbff1b_0
libwebp                   1.3.2                hbc33d0d_0
libwebp-base              1.3.2                h2bbff1b_0
llvmlite                  0.41.1                   pypi_0    pypi
lz4-c                     1.9.4                h2bbff1b_0
markdown                  3.4.1            py38haa95532_0
markupsafe                2.1.4                    pypi_0    pypi
mkl                       2023.1.0         h6b88ed4_46358
mkl-service               2.4.0            py38h2bbff1b_1
mkl_fft                   1.3.8            py38h2bbff1b_0
mkl_random                1.2.4            py38h59b6b97_0
mpc                       1.1.0                h7edee0f_1
mpfr                      4.0.2                h62dcd97_1
mpir                      3.0.0                hec2e145_1
mpmath                    1.3.0            py38haa95532_0
multidict                 6.0.4            py38h2bbff1b_0
natsort                   8.4.0                    pypi_0    pypi
networkx                  3.1              py38haa95532_0
numba                     0.58.1                   pypi_0    pypi
numpy                     1.24.4                   pypi_0    pypi
numpy-base                1.24.3           py38h8a87ada_1
oauthlib                  3.2.2            py38haa95532_0
opencv-python-headless    4.9.0.80                 pypi_0    pypi
openjpeg                  2.4.0                h4fc8c34_0
openssl                   3.0.13               h2bbff1b_0
opt_einsum                3.3.0              pyhd3eb1b0_1
packaging                 23.2                     pypi_0    pypi
pillow                    10.0.1           py38h045eedc_0
pip                       23.3.1           py38haa95532_0
platformdirs              3.10.0           py38haa95532_0
pooch                     1.7.0            py38haa95532_0
protobuf                  4.25.2                   pypi_0    pypi
pyasn1                    0.5.1                    pypi_0    pypi
pyasn1-modules            0.3.0                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0
pygments                  2.17.2                   pypi_0    pypi
pyjwt                     2.4.0            py38haa95532_0
pyopenssl                 23.2.0           py38haa95532_0
pyqt6                     6.6.1                    pypi_0    pypi
pyqt6-qt6                 6.6.1                    pypi_0    pypi
pyqt6-sip                 13.6.0                   pypi_0    pypi
pyqtgraph                 0.13.3                   pypi_0    pypi
pyreadline                2.1                      py38_1
pysocks                   1.7.1            py38haa95532_0
python                    3.8.18               h1aa4202_0
pytorch                   2.2.0           py3.8_cuda11.8_cudnn8_0    pytorch
pytorch-cuda              11.8                 h24eeafa_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pyyaml                    6.0.1            py38h2bbff1b_0
qtpy                      2.4.1                    pypi_0    pypi
re2                       2022.04.01           hd77b12b_0
requests                  2.31.0           py38haa95532_0
requests-oauthlib         1.3.0                      py_0
roifile                   2023.5.12                pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
setuptools                68.2.2           py38haa95532_0
six                       1.16.0             pyhd3eb1b0_1
sqlite                    3.41.2               h2bbff1b_0
superqt                   0.6.1                    pypi_0    pypi
sympy                     1.12             py38haa95532_0
tbb                       2021.8.0             h59b6b97_0
tensorboard               2.10.0           py38haa95532_0
tensorboard-data-server   0.6.1            py38haa95532_0
tensorboard-plugin-wit    1.8.1            py38haa95532_0
tensorflow                2.3.0           mkl_py38h8c0d9a2_0
tensorflow-base           2.3.0           eigen_py38h75a453f_0
tensorflow-estimator      2.6.0              pyh7b7c402_0
termcolor                 2.1.0            py38haa95532_0
tifffile                  2023.7.10                pypi_0    pypi
tk                        8.6.12               h2bbff1b_0
torchaudio                2.2.0                    pypi_0    pypi
torchvision               0.17.0                   pypi_0    pypi
tqdm                      4.66.1                   pypi_0    pypi
typing_extensions         4.9.0            py38haa95532_1
urllib3                   2.2.0                    pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
werkzeug                  2.3.8            py38haa95532_0
wheel                     0.41.2           py38haa95532_0
win_inet_pton             1.1.0            py38haa95532_0
wrapt                     1.14.1           py38h2bbff1b_0
xz                        5.4.5                h8cc25b3_0
yaml                      0.2.5                he774522_0
yarl                      1.9.3            py38h2bbff1b_0
zipp                      3.17.0           py38haa95532_0
zlib                      1.2.13               h8cc25b3_0
zstd                      1.5.5                hd43e919_0

**Comment by carsen-stringer** [comment]
you could try the 10.2 install, details here: https://pytorch.org/get-started/previous-versions/

but unfortunately the GPU might be too old. You could also run through google colab. Going to close this issue but will reopen if there are more questions.
